<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntersectionCircleCircleWindowClass</class>
 <widget class="QMainWindow" name="IntersectionCircleCircleWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IntersectionCircleCircleWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>806</height>
     </rect>
    </property>
    <property name="title">
     <string>显示窗口</string>
    </property>
    <widget class="CMyLabel" name="lbl_Window">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_ReadImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>打开图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_FitImage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>自适应</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>591</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>坐标灰度信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ClearWindow">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>清空窗口</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckb_ShowObj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缩放时显示图形</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Hide">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>650</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>541</width>
      <height>801</height>
     </rect>
    </property>
    <property name="title">
     <string>操作窗口</string>
    </property>
    <widget class="QTabWidget" name="tabWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>501</width>
       <height>761</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>输入</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>471</width>
         <height>531</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>圆A</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>401</width>
             <height>211</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Row:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Row1_CircleA">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Col:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Col1_CircleA">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Radius:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Radius_CircleA">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>圆B</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>401</width>
             <height>211</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Row:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Row1_CircleB">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Col:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Col1_CircleB">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Radius:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="dspb_Radius_CircleB">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>999999999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>590</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="pb_Run">
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_IntersectCircleCircleCT">
          <property name="text">
           <string>运行时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_15">
      <attribute name="title">
       <string>结果</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>461</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Row1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Row1_ResultCircleCircle"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Col1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Col1_ResultCircleCircle"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Row2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Row2_ResultCircleCircle"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Col2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Col2_ResultCircleCircle"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyLabel</class>
   <extends>QLabel</extends>
   <header>cmylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
